wandb/wandb_run.py

```diff
@@ -415,11 +415,26 @@ def save(self, id=None, program=None, summary_metrics=None, num_retries=None, ap
         return upsert_result
 
     def use_artifact(self, artifact, type=None, aliases=None):
+        """ Declare an artifact as an input to a run, call `download` or `file` on \
+        the returned object to get the contents locally.
+
+        Args:
+            artifact_or_name (str or Artifact): An artifact name. May be prefixed with entity/project. Valid names
+                can be in the following forms:
+                    sequence_name:version
```

Comment:
Should we omit `sequence` from these? I'm worried this could be confusing since we never explicitly mention sequences anywhere. I think in the UI and CLI we use:

- `type` for `ArtifactType`
- `name` for `ArtifactSequence`
- `version` for `Artifact`

Reply:
@shawnlewis I just changed all sequence_name instances to name, but it's a bit confusing that currently calling `.name` on an artifact returns `name:version`...

Reply:
We should probably make `name` return just the name part, and maybe have another API to fetch aliases.

Reply:
(not necessarily now, but eventually)

Reply:
I think that's best for now. We still don't have good lingo here.